Sources/StreamChat/Audio/AudioPlayer/AudioPlaying.swift

```diff
@@ -120,48 +120,12 @@ open class StreamAudioPlayer: AudioPlaying, AppStateObserverDelegate {
         subscriber.audioPlayer(self, didUpdateContext: context)
     }
 
-    open func loadAsset(from url: URL) {
-        /// We are going to check if the URL requested to load, represents the currentItem that we
-        /// have already loaded (if any). In this case, we will try either to resume the existing playback
-        /// or restart it, if possible.
-        if let currentItem = player.currentItem?.asset as? AVURLAsset,
-           url == currentItem.url,
-           context.assetLocation == url {
-            /// If the currentItem is paused, we want to continue the playback
-            /// Otherwise, no action is required
-            if context.state == .paused {
-                play()
-            } else if context.state == .stopped {
-                /// If the currentItem has stopped, we want to restart the playback. We are replacing
-                /// the currentItem with the same one to trigger the player's observers on the updated
-                /// currentItem.
-                player.replaceCurrentItem(with: .init(asset: currentItem))
-                play()
-            }
-
-            /// This case may be triggered if we call ``loadAsset`` on a player that is currently
-            /// playing the URL we provided. In this case we will Inform the delegate about the
-            /// current state.
-            notifyDelegates()
-
-            return
-        }
-
-        /// We call stop to update the currently set delegate that the playback has been stopped
-        /// and then we remove the current item from the player's queue.
-        stop()
-        player.replaceCurrentItem(with: nil)
-
-        updateContext {
-            $0.state = .loading
-            $0.assetLocation = url
-        }
-        let asset = AVURLAsset(url: url)
+    public func loadAsset(from url: URL) {
```

Comment:
This should be open right? Otherwise customers who are already overriding the function without the options argument it will not compile

Reply:
Great catch! Thanks